### DISCUSSION TITLE
EF Core extensions

### DISCUSSION THREAD

**Discussion by mweel1** [jasontaylordev/CleanArchitecture#854] [General]
Is there an easy way to push forward an extension library (specifically) https://github.com/borisdj/EFCore.BulkExtensions.

Since DBContext is exposed through the interface, these extensions aren't getting picked up.

I really don't want to just create a bunch of proxy methods to this class if I don't have to.  Any ideas?

**Comment by MMF** [comment]
This feature is now integrated in EF Core 7.0
https://learn.microsoft.com/en-us/ef/core/what-is-new/ef-core-7.0/whatsnew#executeupdate-and-executedelete-bulk-updates
